src/test/java/org/jenkinsci/plugins/ewm/steps/DiskAllocationStrategyTest.java

```diff
@@ -35,22 +39,26 @@ public void tearDown() {
     }
 
     @Test
-    public void useAllocationStrategyInPipeline() throws Exception {
+    @Parameters({"fastestWriteSpeed()",
+                 "[$class: 'FastestWriteSpeedStrategy']"})
+    public void useAllocationStrategyInPipeline(String strategy) throws Exception {
```

Comment:
I've added tests for both old and new implementation by using the `Parameters` annotation from `pl.pragmatists.JUnitParam`